Makefile

```
@@ -10,13 +10,13 @@ release: tidy test release-nix release-win release-mac
 
 define build
 echo "  >  Building binary ..."; \
-cd pkg/$(1); \
+cd $(1); \
 go build -o ../../$(1) -ldflags="$(BUILD_FLAGS)";
```

```suggestion
go build -o ../dist/$(1) -ldflags="$(BUILD_FLAGS)";
```
^ that along with `mkdir dist && touch dist/.gitkeep` perhaps

---

mind if we do that in a follow-up? Seems unrelated to the current change set